x-pack/plugins/secops/public/components/page/hosts/host_summary/index.test.tsx

```diff
@@ -70,10 +74,12 @@ describe('Host Summary Component', () => {
       );
       const wrapper = mountWithIntl(
         <ReduxStoreProvider store={store}>
-          <DragDropContextWrapper>{draggable}</DragDropContextWrapper>
+          <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
+            <DragDropContextWrapper>{draggable}</DragDropContextWrapper>
+          </ThemeProvider>
         </ReduxStoreProvider>
       );
-      expect(wrapper.text()).toBe('yesterday');
+      expect(wrapper.find('FormattedRelative')).toHaveLength(1);
```

Comment:
@FrankHassanabad In effort to simplify things I changed this to just test for the presence of a `FormattedRelative` element. That was the intent of the test after all, so no need to deal with actual relative timestamps. When that is necessary though, [this](https://github.com/yahoo/react-intl/wiki/Testing-with-React-Intl#relativedate-advanced-uses-injectintl) is one way to do it.